src/reducers/todosReducer.js

```diff
@@ -1,34 +1,64 @@
-import actionConstants from '../constants/action_constants'
-import toDoReducer from './toDoReducer'
+import actionConstants from '../constants/action_constants';
 
 function getId(todos){
-  return todos.reduce((maxId, todo) => {
-    return Math.max(todo.id, maxId)
-  }, -1) + 1
+  return todos.ids.reduce((maxId, id) => {
+    return Math.max(id, maxId);
+  }, -1) + 1;
 }
 
-export default function toDosReducer(todos = [{id: 1, text: "Foo", completed: false}], action){
+/*
```

Comment:
I think I understand this. Would you say there's not really a benefit to having this use the toDoReducer either?
I need to read a few more examples on how this is useful.